ydb/docs/en/core/integrations/goose.md

```diff
@@ -0,0 +1,128 @@
+
+If connecting to a local YDB docker container, the connection string should look like:
+
+```
+grpc://localhost:2136/local?go_query_mode=scripting&go_fake_tx=scripting&go_query_bind=declare,numeric
+```
+
+Further examples of calling `goose` commands will contain exactly this connection string.
+
+## Directory with migration files
+
+Let's create a migrations directory and then all `goose` commands should be executed in this directory:
+
+```
+$ mkdir migrations && cd migrations
+```
+
+## Adding a migration file
+
+The migration file can be generated using the `goose create` command:
+
+```
+$ goose ydb "grpc://localhost:2136/local?go_query_mode=scripting&go_fake_tx=scripting&go_query_bind=declare,numeric" create 00001_create_first_table sql
+2023/12/15 05:22:48 Created new file: 20231215052248_00001_create_table_users.sql
+```
+
+This means that the tool has created a new migration file `<timestamp>_00001_create_table_users.sql` where we can record the steps to change the schema for the YDB database, which is accessible through the corresponding connection string.
+
+So, after executing the goose create command, a migration file `<timestamp>_00001_create_table_users.sql` will be created with the following content:
+
+```
+-- +goose Up
+-- +goose StatementBegin
+SELECT 'up SQL query';
+-- +goose StatementEnd
+
+-- +goose Down
+-- +goose StatementBegin
+SELECT 'down SQL query';
+-- +goose StatementEnd
+```
+
+This migration file structure helps keep the instructions that lead to the next version of the database in context. It is also easy, without unnecessary distractions, to write instructions that roll back a database change.
+
+The migration file consists of two sections. The first is `+goose Up`, an area where we can record the migration steps. The second is `+goose Down`, an area in which we can write a step for invert making changes for the `+goose Up` steps. `Goose` carefully inserted placeholder queries:
+
+```
+SELECT 'up SQL query';
+```
+
+And
+
+```
+SELECT 'down SQL query';
+```
+
+so that we can instead enter essentially the migration requests themselves.
+
+Let's edit the migration file `<timestamp>_00001_create_table_users.sql` so that when applying the migration we create a table of the structure we need, and when rolling back the migration we delete the created table:
```

Comment:
```suggestion
Let's edit the migration file `<timestamp>_00001_create_table_users.sql` so that when applying the migration we create a table with necessary schema, and when rolling back the migration we delete the created table:
```